components/CancellationList.vue

```diff
@@ -142,7 +144,16 @@ export default {
         sortable: false,
       },
     ],
+    //SnackBar Notification Data
+    snackBarData: {
+      open: false,
+      text: '',
+      onClosed: ()=>{}
```

Comment:
techincally this event isn't `closed` it's when the close button is clicked. We should update this to `closeBtnClicked` in the snackbar component and rename this event handler `handleSnackBarCloseBtnClicked`